### DISCUSSION TITLE
Need clarification on issue #3997

### DISCUSSION THREAD

**Discussion by DmitrySenin** [kedacore/keda#4442] [Q&A / Need Help] [answered]
Hello,
I'm interested in the behaviour described in https://github.com/kedacore/keda/issues/3997 and I see the issue is closed. Moreover, according to the roadmap, it's done
But I cannot see it's reflected in documentation. Can anybody help to understand how to enable the described behaviour?

**Comment by JorTurFer** [comment]
Hi @DmitrySenin ,
The feature was merged and released some time ago. 
Currently, the Kafka scaler limits the max count to the partition count automatically if `allowIdleConsumers` is false (which is the default value). You can find more information about it in the [scaler docs](https://keda.sh/docs/2.10/scalers/apache-kafka/)
![image](https://user-images.githubusercontent.com/36899226/231592739-07c2d892-4d6b-4ba5-a924-add51798aad4.png)

**Reply by JorTurFer** [reply]
but, take into account that this only applies if you have only a kafka trigger in your ScaledObject, if you have other triggers too, they can trigger the scaling out over partition count as the partition limit is something internal of kafka scaler and it isn't propagated to other triggers

**Reply by DmitrySenin** [reply]
Hi @JorTurFer ,
It's not the behaviour described in the ticket. What I'm looking for is to limit the max count to the number of partitions with lag not just the total number of partitions

**Reply by JorTurFer** [reply, marked as answer]
oh, sorry, I read it wrongly :( You are right
The issue wasn't solved, was autoclosed but it wasn't implemented
I reopened it as you were willing to contribute 😄

**Reply by DmitrySenin** [reply]
No worries :) I'm already working on the implementation of the behaviour
